tests/phpunit/tests/query/cacheResults.php

```diff
@@ -710,7 +710,7 @@ public function test_query_cache_different_fields() {
 		$query2->query( $args );
 		$queries_after = get_num_queries();
 
-		$this->assertSame( $queries_before, $queries_after );
+		$this->assertSame( 1, $queries_after - $queries_before );
```

Comment:
This has changed now,  because we don't call `_prime_post_caches` on id=>parent. This extra query is expected, as it is priming post objects.